lib/index.js

```diff
@@ -41,7 +41,7 @@ function lookup (uri, opts) {
 
   opts = opts || {};
 
-  var parsed = url(uri);
+  var parsed = url(uri, null, (opts.path ? opts.path : null));
```

Comment:
Are both checks `(opts.path ? opts.path : null)` and `(optsPath || '')` mandatory?